src/main/java/io/github/linuxforhealth/core/config/ConverterConfiguration.java

```diff
@@ -45,11 +46,19 @@ public class ConverterConfiguration {
   private String additionalResourcesLocation;
 
   private ConverterConfiguration() {
-    try {
+    List<FileLocationStrategy> subs = Arrays.asList(new ConfigDirectoryLocationStrategy(),
+            new ClasspathLocationStrategy());
+    FileLocationStrategy strategy = new CombinedLocationStrategy(subs);
+    initializeConfiguration(strategy);
+  }
+
+  public ConverterConfiguration(String configFolderPath) {
```

Comment:
Maybe just some comments here explaining the two constructors and the expected use cases.